### PR TITLE
Don't trap-delete directory on failure.

### PR DESCRIPTION
**Pull request by tomberek**
This may delete a record of CSRs or other things inside the certs/clients/SAFE_NAME directory.

### CODE REVIEW

File: sign-csr [hunk 1]
```diff
@@ -72,8 +72,8 @@ else
     SAFE_NAME=$(echo "$CSR_CERT_CN" | sed 's/\*/star/g' | sed 's/[^A-Za-z0-9-]/-/g')
 fi
 
-if [ -d "certs/clients/$SAFE_NAME" ]; then
-    echo -e "$ERR Configuration already exist for '$CSR_CERT_CN' ($SAFE_NAME), exiting."
+if [ -d "certs/clients/$SAFE_NAME"/"$SAFE_NAME".crt ]; then
```
Comment (cgzones) [inline sign-csr]: `if [ -f "certs/clients/$SAFE_NAME/$SAFE_NAME".crt ]; then`

File: sign-csr [hunk 2]
```diff
@@ -91,15 +91,15 @@ openssl rsa -check \
             -passin env:CA_PASS \
             -noout
 
-trap 'rm -Rf "certs/clients/$SAFE_NAME"' 0
-mkdir "certs/clients/$SAFE_NAME"
+echo -e "$NOTE Creating client directory certs/clients/$SAFE_NAME"
+mkdir -p "certs/clients/$SAFE_NAME"
```
Comment (cgzones) [inline sign-csr:95]: no need for `-p`, this directory is created at the ca initialization
https://github.com/cgzones/easy-ca/blob/sign-csr_dev/functions#L107

Comment (tomberek) [reply sign-csr:95]: mkdir will error if the dir already exits

File: sign-csr (hunk 1 above)
Comment (tomberek) [reply sign-csr]: yep!

**Comment by cgzones** [comment]
Thanks

**Pull request merged**